docs/metrics/user_metrics.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Metrics
+sidebar_label: User Accounting Metrics
```

Comment:
Thats a long sidebar label.  It will be under the metrics category, so perhaps just "Users" or "User Accounting"?

Also, the title is the title on the page, so you probably want it to be "User Accounting Metrics"

Finally, the slug should change:

/metrics/users or something